src/Caliburn.Micro.Platform/Caliburn.Micro.Platform.csproj

```diff
@@ -79,5 +87,14 @@
     <Compile Include="ViewModelLocator.cs" />
   </ItemGroup>
 
+  <!--<PropertyGroup Condition="'$(TargetFramework)' == 'net6.0-android'">
+    <UseMaui>true</UseMaui>
```

Comment:
Why not just remove the commented out code

Reply:
Yes that should be removed. I like to green field the code while working guess I should use git and commit more often :)